data/mods/Medieval_Stuff/tools.json

```diff
@@ -0,0 +1,543 @@
+[
+  {
+    "id": "arming_sword",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "arming sword",
+    "description": "This is a classic medieval sword, just the right size to use one-handed.",
+    "price": 100000,
+    "material": "steel",
+    "flags" : ["CHOP", "DURABLE_MELEE", "SHEATH_SWORD"],
+    "techniques": ["RAPID", "WBLOCK_1"],
```

Comment:
Rapid on swords this big doesn't make much sense (katana shouldn't have it either). Broadsword doesn't have it and it's smaller than those.
Swords of this size are too heavy to be used for rapid strikes.